src/index.html

```diff
@@ -22,6 +22,489 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!--#region header-->
+    <header class="header">
+      <a
+        href="#index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo__img"
+        />
+      </a>
+
+      <!--#region nav-->
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the liks
```suggestion
          </li>
          
          <li class="nav__item">
```